shared/common-adapters/back-button.stories.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import * as React from 'react'
+import {storiesOf, action} from '../stories/storybook'
+import {Box2, BackButton} from '.'
+import {globalColors} from '../styles'
+
+const defaultProps = {
+  badgeNumber: 0,
+  onClick: action('onClick'),
+  onPress: action('onPress'),
+}
+
+const load = () => {
+  storiesOf('Common', module).add('Back Button', () => (
+    <Box2 direction="vertical">
+      <BackButton {...defaultProps} />
+      <BackButton {...defaultProps} badgeNumber={5} />
```

Comment:
Should I see a badge?
<img width="128" alt="image" src="https://user-images.githubusercontent.com/11968340/42908834-feea13f2-8aaf-11e8-8405-b958b008834e.png">

Reply:
The back button only shows badges on mobile so nope

Reply:
unless that is mobile

Reply:
oh that's not mobile. cool :+1: